### PR TITLE
Fix faust memory create

### LINKED ISSUE #460: Rack segfaults when attempting to load a plugin generated via faust
G'day,

Rack is instantly crashing when trying to load a plugin that was built by erbb from faust sources - such as the flanger example.

I have fixed it locally [here](https://github.com/ohmtech-rdi/eurorack-blocks/compare/main...jazamatronic:eurorack-blocks:rack_segfault?expand=1#diff-3fea5fd94ce514fc3d31e7eb873a06e9d46e053cee88f8cefcd4534b8810d2c8) by initializing dsp memory during the init.

I can create a pull request if you'd like?

Cheers

### PR DESCRIPTION
**Pull request by ohmtech-rdi**
Fix missing internal buffers allocation by calling `memoryCreate` before the DSP instance is inited, following the [documentation](https://faustdoc.grame.fr/manual/architectures/#defining-and-using-a-custom-memory-manager) on the new `-mem` option.

The `memoryCreate` function is a new function, but which must be called, in order for the DSP to correctly instantiate.

To keep compatibility with pre-2.41, we are checking whether the function exists or not at build time and call it if available.

- Fixes #460

### CODE REVIEW

**Comment by ohmtech-rdi** [comment]
@jazamatronic Please evaluate

File: .github/workflows/windows_2019.yml [hunk 1]
```diff
@@ -83,8 +83,8 @@ jobs:
       - name: Check max
         run: grep "Loaded ErbPluginMax" $USERPROFILE/Documents/Rack2/log.txt
 
-  software_faust:
-    name: Software Faust
+  software_faust_2_37_3:
```
Comment (jazamatronic) [inline .github/workflows/windows_2019.yml:86]: I can't comment on the windows build system

File: build-system/erbb/generators/faust/code_template.hpp [hunk 2]
```diff
@@ -79,6 +99,7 @@ void  %module.name%::init ()
    mydsp::fManager = &mem;
 
    mydsp::classInit (erb_SAMPLE_RATE);
+   call_faust_mydsp_memoryCreate_if_exists (dsp, 0);
```
Comment (jazamatronic) [inline build-system/erbb/generators/faust/code_template.hpp:102]: This looks good and works with the 2.50.6 build of Faust I have installed.
The VCV rack plugin now builds, loads and functions as expected.  I haven't had a chance to test it on the hardware but the daisy binary builds fine.

**Review by jazamatronic — APPROVED** [review]
Comments in line below - but in general I can't comment on the windows build changes, but have validated that the VCV rack plugin changes work in Rack2 (2.1.2 and 2.2.0) with Faust 2.50.6.
Daisy builds but hasn't been tested on hardware.

**Comment by jazamatronic** [comment]
G'day @ohmtech-rdi I'm new to github reviews, I hope I've done the right thing.  In any case it looks good.  Let me know if you need anything else.

**Comment by ohmtech-rdi** [comment]
Hi @jazamatronic You did right! Thanks a lot for your review.

**Pull request merged**